testgrid/tgrun/pkg/instances/k8s119_nameserver.go

```diff
@@ -0,0 +1,30 @@
+package instances
+
+import (
+	kurlv1beta1 "github.com/replicatedhq/kurl/kurlkinds/pkg/apis/cluster/v1beta1"
+	"github.com/replicatedhq/kurl/testgrid/tgrun/pkg/scheduler/types"
+)
+
+func init() {
+	RegisterInstance(
+		types.Instance{
+			InstallerSpec: types.InstallerSpec{
+				Kubernetes: kurlv1beta1.Kubernetes{
+					Version: "1.19.3",
+				},
+				Weave: &kurlv1beta1.Weave{
+					Version: "2.6.4",
+				},
+				Rook: &kurlv1beta1.Rook{
+					Version: "1.0.4",
```

Comment:
do we need rook here?
just want to keep tests minimal (and thus theoretically faster), and rook 1.0.4 is definitely tested elsewhere

Reply:
No, I'll remove